### DISCUSSION TITLE
手机测试

### DISCUSSION THREAD

**Discussion by su2569** [share121/inter-knot#2069] [其他]
手机有一个bug，下滑时会自动弹到上面

**Comment by 1luik** [comment]
你试试电脑模式

**Comment by su2569** [comment]
照样


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月15日 14:37 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 手机测试 (Discussion #2069) |

你试试电脑模式

**Comment by su2569** [comment]
之后无法连接了



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月15日 14:37 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 手机测试 (Discussion #2069) |

你试试电脑模式
